userspace/libscap/scap_savefile.c

```diff
@@ -1091,6 +1126,28 @@ scap_dumper_t *scap_memory_dump_open(scap_t *handle, uint8_t* targetbuf, uint64_
 	return res;
 }
 
+//
+// Create a dumper with an internally managed buffer
+//
+scap_dumper_t *scap_managedbuf_dump_create(scap_t *handle)
+{
+	scap_dumper_t *res = (scap_dumper_t *)malloc(sizeof(scap_dumper_t));
+	if(res == NULL)
+	{
+		snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE, "scap_managedbuf_dump_create memory allocation failure (1)");
+		return NULL;
+	}
+
+	res->m_f = NULL;
+	res->m_type = DT_MANAGED_BUF;
+	const size_t targetbufsize = 3 * 1024 * 1024;
+	res->m_targetbuf = (uint8_t *)malloc(targetbufsize);

```

Comment:
```suggestion
	res->m_targetbuf = (uint8_t *) malloc(sizeof(uint8_t) * targetbufsize);
```
nit-picking here

Reply:
`uint8_t` is guaranteed to be 8 bits, so it's absolutely not needed. And if I add the extra space and format with the current settings it gets removed